.github/workflows/ci-release.yml

```
@@ -1,11 +1,12 @@
 name: ci-release
 
-# Trigger on push events to main (i.e. merges), pushing new semantic version
-# tags, all PRs, and manual triggers
+# Run this workflow on push events (i.e. PR merge) to main or release branches,
+# push events for new semantic version tags, all PRs, and manual triggers.
 on:
   push:
     branches:
       - main
+      - "v[0-9]+.x"
```

maybe also
```suggestion
      - "v[0-9]+.x"
      - "v[0-9]+.[0-9]+.x"
```
stupid formatting question, does 0-9 just mean any number, or literally 0 through 9?

---

- `[0-9]` means any digit
- `+` means 1 or more

hmm I think we only need `"v[0-9]+.[0-9]+.x"` if we intend on maintaining release branches for minor versions. In other words:

- `v1.x`
- `v1.0.x`
- `v1.1.x`
- `v1.2.x`

but I thought we were only planning on maintaining release branches per major version